content/F2017/F17-event-dinner-with-professors.md

```diff
@@ -0,0 +1,12 @@
+title: Dinner with the Professors
+Date: 2017-11-08
+Category: Events
+Tags: events, social
+Slug: dinner-with-professors-F17
+Author: Ruofan Luo
+Summary: Join WiCS for our Dinner with the Professors event!
+
+Come meet your profs! We know you've seen them in lecture, but what are they like outside of class? Come socialize with CS professors over a casual dinner. Ask them anything CS or research related, or just get to know them!
```

Comment:
Lines need to be 80 chars long.

Reply:
* put the date and time (Monday, November 20, 2017 6pm-8pm)
* remember to say "This is a women-only event." on ALL our events (unless otherwise specified) and link to to our definition of women-only on the website (http://wics.uwaterloo.ca/faq/)
* At the end add: "We will be serving Boa with options for dietary restrictions. This is the first of our two Mentorship Track events, the other being Speed Mentoring (link) on November 21, 2017. Participants must sign-up in advance (link to form/eventbrite which I will provide you with soon) and attending both events will give you first pick of our new T-Shirts!"

Reply:
Again add the 

Who 
What where when why